contrib/ast-db-manage/config/versions/2b7c507d7d12_add_queue_log_option_log_restricted_.py

```diff
@@ -24,11 +24,7 @@
 def upgrade():
     # Create the new enum
```

Comment:
This line should go to. Other migrations use this comment (you don't have to though):

```text
# ast_bool_values have already been created, so use postgres enum object
# type to get around "already created" issue - works okay with mysql
```

Reply:
Done